ORBIT/phases/design/mooring_system_design.py

```diff
@@ -161,15 +161,15 @@ def calculate_line_length_mass(self):
         """
 
         # Add extra fixed line length for drag embedments
-        if self.anchor_type == "Drag Embedment":
+        if any(w in self.anchor_type.lower() for w in ["drag", "embedment"]):
```

Comment:
It seems like some input standardization might be needed to reduce some of the work in this PR

Reply:
Went back to old logic, but changed self.anchor_type to "title()" case when first instantiated.